dnsutil/util.go

```diff
@@ -0,0 +1,91 @@
+package dnsutil
```

Comment:
I think you need to remove empty line between doc and package. You can check doc running ```godoc github.com/miekg/dns/dnsutil``` or use http-server of godoc (assuming you edit package inside GODOC).  Common practice to use "Package pkgname" as start of first sentence but godoc would work without it too.